mindmeld/components/classifier.py

```diff
@@ -403,6 +403,9 @@ def _get_model_config(loaded_config=None, **kwargs):
         Returns:
             ModelConfig: The model configuration corresponding to the provided config name
         """
+        if 'params' in loaded_config and 'params' in kwargs:
+            merged_params = {**loaded_config['params'], **kwargs['params']}
+            kwargs['params'] = merged_params
```

Comment:
would a one-liner work here?
```suggestion
            kwargs['params'] = {**loaded_config['params'], **kwargs['params']}
```

Reply:
Refactored!